plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -909,6 +915,24 @@ private void populateSplits(File war) throws IOException {
         }
         throw new IOException("no jenkins-core-*.jar found in " + war);
     }
+
+    private List<String> removeSplitsBasedOnJDK(List<String> splits, VersionNumber jdkVersion) {
+        List<String> filterSplits = new LinkedList();
+        for (String split : splits) {
+            String[] tokens = split.trim().split("\\s+");
+            if (tokens.length == 4 ) { // We have a jdk field in the splits file
+                if (jdkVersion.isNewerThan(new VersionNumber(tokens[3]))) {
```

Comment:
Please use https://github.com/jenkinsci/lib-version-number/blob/master/src/main/java/io/jenkins/lib/versionnumber/JavaSpecificationVersion.java . Otherwise there is a risk that versions are compared incorrectly if somebody creates a Java 8 - only plugin (e.g. for Java Web Start)

Reply:
Done